### ISSUE TITLE
does hea file contain the ecg data?

### ISSUE THREAD

**Issue by Guyume** [tom-beer/Arrhythmia-CNN#1]
Hi tom，I find that you save fragment of hea file as data. But I plot it and find that the wave of hea file looks a little different with wave plotted by physioBank ATM. Is it ok or are they the same thing?

**Comment by tom-beer** [comment]
Unfortunately, I am not familiar with the PhysioBank ATM.
The contents of the 'data' folder was downloaded from here:
https://physionet.org/content/mitdb/1.0.0/
As my understanding goes, this is the dataset used by the authors in the paper.

**Comment by Guyume** [comment]
@tom-beer Thx for your reply. 
I am reproducing the paper's algorithm too. But there're something wired when I do the job. The wiredest one is that the 17 classes contain both beat annotation and rhythm annotation like  the Atrial premature beat and  Normal sinus rhythm. It not easy for data building. How do you think this problem when you face it?

**Comment by tom-beer** [comment]
That's right, most classes in the database contain two labels - rhythm and beat.
The database building was not straightforward for us, too.
For each of the classes from the paper, we decided to find the most indicative label in the dataset. For some it was the beat, for others it was the rhythm. 
The dictionary 'num_labels_dict' has in the comments the source of each label.
This was the only way we could think of to build the database, but having no one to consult with, it may be wrong. 
If you have any thoughts on this issue, please share!

**Comment by Guyume** [comment]
yeah, I think so. And I have done some statistical work for the data, we can have a discuss if possible.
Another question, there are only 12 key in the dict 'num_labels_we_have', Is there any reason for this?

**Comment by tom-beer** [comment]
Sorry for the late reply.
We built our dataset according to the 13-label classification problem (Table 1).
In addition, we removed the class of ventricular flutter due to low number of training examples.
In total, the dataset has 12 classes.
